### ISSUE TITLE
Tab order is not logical between the object menu and main content in Vertical Nav tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1923]
URL: http://clarity-ks.surge.sh/vertical-nav

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Vertical Nav tab.
3. Use tab key to access all interactive elements.

Actual Results:
The tab order is not logical. 
It points to main content page, then points to the object menu at left of main content page.
For more details, please refer to “VerticalNav-TabOrderIsNotLogical.png”.

Expected Results:
The tab order should keep from left to right, and here it should point to the object menu before point to the main content page.

Violation of Section 508 Standards: 
1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind and disable with hand.

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/32d688b6-e035-4e11-9245-fa6b7aeb7206)

NOTE: We need to understand if this is an implementation issue or an issue with Clarity. If an implementation issue, this is probably a problem on the website as well.

**Comment by gnomeontherun** [comment]
Might be just an issue with our demo app, or the nav is after the main content (for some reason?).

**Comment by Jinnie** [comment]
The nav IS after the main content. This is a common patter I am observing on many places in our demo code. It probably originates from the times we didn't use "Skip to content" links. I'd appreciate any comments about the historical reasons behind this.

**Comment by Jinnie** [comment]
Updated for both sidenav and vertical nav, in dev-app, ks-app, website.
